build.gradle.kts

```diff
@@ -3,21 +3,19 @@ import org.gradle.api.tasks.testing.logging.TestExceptionFormat
 import org.gradle.api.tasks.testing.logging.TestLogEvent
 
 plugins {
-    id("java")
-    id("java-library")
-    id("maven-publish")
+    java
+    `java-library`
+    `maven-publish`
+    signing
```

Comment:
Why not unified?

Reply:
sticking to the official conventions also used in the Gradle documentation